src/modules/fancyzones/editor/FancyZonesEditor/App.xaml.cs

```diff
@@ -66,5 +73,94 @@ private void OnStartup(object sender, StartupEventArgs e)
             overlay.Show();
             overlay.DataContext = foundModel;
         }
+
+        private void OnUnhandledException(object sender, UnhandledExceptionEventArgs args)
+        {
+            var fileStream = File.OpenWrite("FZEditorCrash.log");
```

Comment:
Let's change from .log to .txt so the users will be able to upload it to GitHub without having to change the file extension.